app/src/main/java/com/sdpteam/connectout/profileList/ProfileListActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.sdpteam.connectout.profileList;
+
+import com.sdpteam.connectout.R;
+import com.sdpteam.connectout.drawer.FilterFragment;
+import com.sdpteam.connectout.utils.WithFragmentActivity;
+
+import android.os.Bundle;
+import android.widget.ToggleButton;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+public class ProfileListActivity extends WithFragmentActivity {
```

Comment:
please explain in documentation that this class toggles between list fragment and filter fragment and why that is useful